sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -685,8 +729,13 @@ def receive_deferred_messages(self, sequence_numbers, **kwargs):
                     self._auto_lock_renewer.register(self, message)
             return messages
 
-    def peek_messages(self, max_message_count=1, **kwargs):
-        # type: (int, Any) -> List[ServiceBusReceivedMessage]
+    def peek_messages(
```

Comment:
could you also help add type hint for the abandon/complete/deadletter/defer_message method? :P

though there're not kwargs there, but I feel since we're adding type hint everywhere, we shall keep consistency